misk/src/main/kotlin/misk/web/MiskWebModule.kt

```diff
@@ -164,6 +165,8 @@ class MiskWebModule(
 
     newMultibinder<ConcurrencyLimiterFactory>()
     if (!config.concurrency_limiter_disabled) {
+      install(ConcurrencyLimitsModule(config))
```

Comment:
is this the way we want to do this?

My understanding of how this works is that this will be configured through the `<service>-<environment>.yaml` config files under `skim.web`, is that right?

Reply:
Correct.